docs/MAINTAINERS.adoc

```diff
@@ -149,6 +156,137 @@ code reviewers are expected to suggest proper channels for contributors to discu
+If a core maintainer does not regularly close pull requests,
+the maintainer should consider whether it is appropriate to remain on the core team
+in light of the commitment to closing pull requests that such membership implies.
+
+===== Providing actionable and justified feedback
+
+Once assigned to a pull request, a core maintainer should make every reasonable effort to drive the pull request to closure in a timely fashion.
+If further action is needed before the pull request can be accepted, this action should be explicitly requested along with the reasoning behind it.
+Contributors are far more likely to successfully complete action items when the reasoning behind the request is explicit and cogent.
+It is perfectly reasonable for the assignee or any other reviewer to ask questions,
+but the ultimate goal of these questions should be to arrive at clear and justified action item(s);
+otherwise, the process can languish for an extended period of time.
+It is the responsibility of the assignee to steer the discussion towards concrete and justified action item(s).
+
+===== Providing confirmation that feedback has been addressed
+
+Once any requested actions have been taken, the assignee should make every reasonable effort to provide explicit confirmation that each action item has been completed.
+This gives contributors positive reinforcement and confidence that their submission is moving forward through the process,
+ultimately making them more likely to complete the process and contribute again.
+If the assignee cannot provide such confirmation in a timely fashion,
+the assignee should consider whether it is appropriate to remain assigned to the pull request
+in light of the commitment to the contributor that such an assignment implies.
+
+===== Negotiating scope
+
+Not every pull request reaches a state of perfection at the end of the review process.
+Sometimes, requests are made that, while justified, represent an additional amount of work that the contributor may not be willing to do.
+In some cases it is critical to complete the additional work, but in others "you aren't gonna need it" (YAGNI).
+In such cases, the assignee should make a good faith effort to negotiate with the contributor to find a reasonable middle ground that is "good enough."
+Failure to negotiate reasonably can often chase contributors away.
+If the additional work is simple enough and the submission is not moving forward,
+the assignee may consider occasionally giving the contributor a lift by completing the additional work,
+though this is not expected in the general case
+and would not be reasonable to the assignee if a large amount of additional work is necessary.
+
+===== Voting process
+
+A pull request can often serve as a catalyst for a discussion in which several possible paths forward are identified.
+When there is no clear consensus among the core maintainers about the path forward,
+the assignee should call for a vote.
```

Comment:
Then I suppose no vote would be called. I do not think it would make sense for a non-assignee to call for a vote. If someone is involved enough with a pull request on a procedural level that they are calling for a vote, they ought to self-assign. Doing one but not the other seems inconsistent and conflicts with the motivation to have a unified point of contact for authors.